cluster/aws/util.sh

```diff
@@ -1016,7 +1029,9 @@ function start-minions() {
       --tags ResourceId=${ASG_NAME},ResourceType=auto-scaling-group,Key=Name,Value=${NODE_INSTANCE_PREFIX} \
              ResourceId=${ASG_NAME},ResourceType=auto-scaling-group,Key=Role,Value=${MINION_TAG} \
              ResourceId=${ASG_NAME},ResourceType=auto-scaling-group,Key=KubernetesCluster,Value=${CLUSTER_ID}
+}
 
+function wait-minions {
```

Comment:
Underlying reason was I'm using these scripts to recreate the auto-scaling group and the minion counting doesn't add up when creating a new auto-scaling group (1 existing minion + 1 new minion = 2 minions > 1 expected minion).  By separating this I'm able to skip the check in my own scripting.  The additional benefit of extracting the function is shorter function bodies ~ cleaner code.